content/influxdb/cloud/notebooks/manage-notebooks.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Manage notebooks
```

Comment:
Thought about splitting View/update into topic and Delete into another topic?...mulling over how we've used "Manage" in other places, for example "Manage tasks" and "Manage dashboards" to also include create...

Reply:
Yeah, I did this because these are so short as I explained in our standup last week. I don't think we should make a bunch of 50 word articles just to be consistent when it doesn't make sense otherwise but am open to other opinions. I'm going to leave for now and we can discuss tomorrow in standup.